docs/src/pages/docs/installation/email_reports.mdx

```diff
@@ -6,8 +6,309 @@ index: 10
 version: 1
 ---
 
-## Scheduling and Emailing Reports
+## Alerts and Reports
+(version 1.0.1 and above)
+
+Users can configure automated alerts and reports to send charts and dashboards to an email recipient or Slack channel.
+
+- Alerts are sent when a specified condition is passed
+- Reports are sent on a specified schedule 
+
+### Turning on Alerts and reports
+Alerts and reports are not turned on by default. They are currently behind a feature flag, and require some additional services and configurations.
+
+#### Requirements:
+
+- `Dockerfile`
+	- webdriver to run a headless browser (for taking screenshots of the charts and dahboards)
+- `docker-compose.yaml`
+	- redis message broker
+	- replacing SQLlite DB with Postgres DB
+	- celery worker
+	- celery beat
+- `superset_config.py`
+	- feature flag turned to True
+	- all configs as outlined in the template below
+- At least one of these is needed to send alerts and reports:
+	- (optional) SMTP server for sending email
+	- (optional) Slack app integration for sending to Slack channels
+ 
+#### Summary of steps to turn on alerts and reporting:
+
+Using the templates below,
+1. Create a new directory and create the Dockerfile
+2. Build the extended image using the Dockerfile
+3. Create the `docker-compose.yaml` file in the same directory
+4. Create a new sub directory called `config`
+5. Create the `superset_config.py` file in the `config` sub directory
+6. Run the image using `docker-compose up` in the same directory as the `docker-compose.py` file
+7. In a new terminal window, upgrade the DB by running `docker exec -it superset-1.0.1-extended superset db upgrade`
```

Comment:
is there a better way to run 
```
superset db upgrade
superset init
superset fab create-admin
```
as part of startup without entering the container?

Reply:
i think the `superset-init` service in the docker-compose workflow should take care of this.